sycl/test/basic_tests/group_async_copy.cpp

```diff
@@ -0,0 +1,150 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.run
+// RUN: %GPU_RUN_PLACEHOLDER %t.run
+// RUN: %CPU_RUN_PLACEHOLDER %t.run
+// RUN: %ACC_RUN_PLACEHOLDER %t.run
+// RUN: env SYCL_DEVICE_FILTER=host %t.run
+
+#include <CL/sycl.hpp>
+#include <iostream>
+#include <typeinfo>
+
+using namespace cl::sycl;
+
+template <typename T> class KernelName;
+
+// Define the number of work items to enqueue.
+const size_t NElems = 32;
+const size_t WorkGroupSize = 8;
+const size_t NWorkGroups = NElems / WorkGroupSize;
+
+template <typename T>
+void initInputBuffer(buffer<T, 1> &Buf, size_t Stride = 0) {
+  auto Acc = Buf.template get_access<access::mode::write>();
+  for (size_t I = 0; I < Buf.get_count(); I += WorkGroupSize) {
+    for (size_t J = 0; J < WorkGroupSize; J++)
+      Acc[I + J] = I + J + ((J % Stride == 0) ? 100 : 0);
```

Comment:
NIT: wow, it's interesting that `Stride` is nil by default. Yet it is a divider here.

Reply:
The initialization was initially written differently, I forgot to remove that default initialization with 0. Will be fixed.